client/web/src/marketing/SurveyUserRatingForm.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+
+import { Button, Checkbox } from '@sourcegraph/wildcard'
+
+import { SurveyRatingRadio } from './SurveyRatingRadio'
+import { Toast } from './Toast'
+
+export interface SurveyUserRatingFormProps {
+    onChange?: (score: number) => void
+    toggleErrorMessage: boolean
+    onContinue: () => void
+    onDismiss: () => void
+    shouldPermanentlyDismiss: boolean
+    toggleShouldPermanentlyDismiss: (value: boolean) => void
+}
+
+export const SurveyUserRatingForm: React.FunctionComponent<SurveyUserRatingFormProps> = ({
+    onChange,
+    toggleErrorMessage,
+    onDismiss,
+    onContinue,
+    shouldPermanentlyDismiss,
+    toggleShouldPermanentlyDismiss,
+}) => (
+    <Toast
+        title="Tell us what you think"
+        subtitle={
+            <span id="survey-toast-scores">How likely is it that you would recommend Sourcegraph to a friend?</span>
+        }
+        cta={
+            <>
+                <SurveyRatingRadio ariaLabelledby="survey-form-scores" onChange={onChange} />
+                {toggleErrorMessage && <div>Please select a score between 0 to 10</div>}
```

Comment:
```suggestion
                {toggleErrorMessage && <div role="alert">Please select a score between 0 to 10</div>}
```

So it notifies screen readers